### ISSUE TITLE
🎨 Improve CSS element selectors

### ISSUE THREAD

**Issue by webcrawls** [kadenscott/bbb#7]
For example, instead of selecting `wrapper_ed1dea` to grab the folder wrapper element, we can select `[aria-label='Servers'] > [class^="wrapper"]`, which should be more resistant to Discord classname changes.
